content/actions/guides/about-continuous-integration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: About continuous integration
+title: About continuous integration and continuous deployment
 intro: 'You can create custom continuous integration (CI) and continuous deployment (CD) workflows directly in your {% data variables.product.prodname_dotcom %} repository with {% data variables.product.prodname_actions %}.'
```

Comment:
Sure in the knowing christopher vincent was the one who discovered this. And if we got him out of the way I'd already be on the next one.  Intiate security on integrity of law of intelect